runelite-client/src/main/java/net/runelite/client/plugins/statusbars/BarRenderer.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2018, Rheon <https://github.com/Rheon-D>
```

Comment:
This should be (c) 2019 and using your info, unless this was in fact written by RheonD